cluster/gce/gci/mounter/Dockerfile

```diff
@@ -16,5 +16,5 @@ FROM ubuntu:xenial
 MAINTAINER vishh@google.com
 
 RUN apt-get update && apt-get install -y netbase nfs-common=1:1.2.8-9ubuntu12 glusterfs-client=3.7.6-1ubuntu1
-
+ADD startmounter.sh /startmounter.sh
 ENTRYPOINT ["/bin/mount"]
```

Comment:
I noticed this will be overwritten by rkt exec command. But I will change to make sure.